src/types/config.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "networks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "type": "string"
+                    },
+                    "rpc": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "deploymentBlock": {
+                        "type": "integer"
+                    },
+                    "watchdogTimeout": {
+                        "type": "integer"
+                    },
+                    "orderBookApi": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "pageSize": {
+                        "type": "integer"
+                    },
+                    "filterPolicy": {
+                        "type": "object",
+                        "properties": {
+                            "defaultAction": {
+                                "$ref": "#/$defs/filterAction"
+                            },
+                            "owners": {
```

Comment:
owners and handlers have the same type. would it be worth it to reference it? 

Actually would be probably be nice to define the FilterPolicy too. This way you will get a Typescript definition for it you can reuse, so you don't need to do things like:
` constructor(config: Config["networks"][number]["filterPolicy"]) {`